src/pages/signup_page/signup_page.scss

```diff
@@ -18,6 +18,7 @@
   flex-grow: 1;
   display: inline;
   text-align: center;
-  input[type="email"] { min-width: 200px; }
-  input[type="password"] { min-width: 200px; }
+  input[type="email"] { min-width: 300px; };
```

Comment:
I don't think the ; after the } is needed

Reply:
true, you just need it after () in scss

Reply:
done and true!